pkg/booleanpolicy/default_policies_test.go

```diff
@@ -2218,7 +2222,7 @@ func (suite *DefaultPoliciesTestSuite) TestImageVerified() {
 		{
 			values:          []string{verifier2},
 			negate:          false,
-			expectedMatches: allImages,
+			expectedMatches: allImages.Difference(set.NewFrozenStringSet("verified_by_2_and_3")),
```

Comment:
I am not sure this is the expectation from the users. The image is verified by `verifier2` already. It is also verified by `verifier3`. Should it alerted if it is already verified by `verified2`?

Reply:
There will be no alert for `verified_by_2_and_3` (it is excluded from the expected matches), which I'd say is expected.

Reply:
thanks for clarification